index.html

```diff
@@ -446,17 +446,25 @@
 
 
-Rainbow-like iridescence effects occur due to interference when a thin refractive film with thickness on the order of the light wavelength is placed on top of a material. To model this, there is assumed to be such a thin-film sitting on top of the base substrate (whether metal or dielectric), parametrized only by its thickness and the film IOR.
-In practice, these effects are most easily incorporated directly into the microfacet BSDFs of both the metal and dielectric-base layers. The currently recommended thin-film model is that of Belcour and Barla [#Belcour2017].
+### Thin-film iridescence
 
-!!! Note: Future work
-   It is currently not physically clear enough how the IOR of the surrounding media affects the thin-film interference, and how the thin-film affects the Fresnel of the adjacent slabs. Moreover, the behavior of the current parametrization is arguably unintuitive, as the effect of the thickness is not obvious. It may be useful to have a more artist-friendly dial of the "iridescence".
+_Iridescence_ is the occurrence of rainbow-like color fringes when a thin dielectric film with thickness on the order of the wavelength of light is placed on top of a material, due to wave interference between the various electromagnetic reflection modes within the film. To model this, there is assumed to be such a thin-film sitting on top of the base substrate (whether metal or dielectric), parametrized only by:
+
+   - the coverage (presence) weight of the film, **`thin_film_weight`**,
+   - the thickness of the film in nanometers, **`thin_film_thickness`**,
+   - the index of refraction (IOR) of the film, **`thin_film_ior`**,
+
+In practice, this wave-optics effect is most easily incorporated directly into the Fresnel factor of the microfacet BSDFs of both the metal and dielectric-base layers. The currently recommended thin-film model is that of Belcour and Barla [#Belcour2017]. The coverage weight functions as a blend between the BSDF with and without the presence of the film, and is necessary since neither the thickness nor the IOR of the film provide a dial of the effect without altering the shape and saturation of the color fringes.
 
+The interference within and resulting reflection from the film will be affected (as described by Belcour and Barla [#Belcour2017]) by the complex IOR of the adjacent media above and below the film, which in general are a statistical mix of metal and dielectric below and coat and ambient medium above. Modeling of the precise effect is implementation-dependent, however as described in the Metal section the Fresnel factor for metal is defined according to the Schlick-based "F82-tint" parametrization, which does not specify the underlying (complex) physical IOR. We suggest here that some reasonable approximation is employed to map the Fresnel factor to effective IOR and absorption coefficients, for example that described by [#Gulbrandsen2014].
```

Comment:
I don't understand:
> The interference within and resulting reflection from the film will be affected [...]

I'm not sure wether it's meant as an ellipsis (_within [the layer]_), or if it's a typo. I think it should be more explicit.
Also, the sentence is fairly long, so I suggest to split it.

Reply:
It's not a typo, I was alluding to the fact that the color fringes arise from interference of the ladder of reflection modes *within* the film,  the sum total effect of which is the color-fringy appearance of the reflection.

If you think it sounds confusing, let's reword. Perhaps:
>The shape and color of the fringe patterns in the reflection from the film will be affected (as described by Belcour and Barla) by the complex IOR of the adjacent media ...

Reply:
Yes, I find this new version much easier to read.

Reply:
Committed in https://github.com/AcademySoftwareFoundation/OpenPBR/pull/167/commits/abf2f7d2cdce3b78de5e284e97dcfc93e7902ad2.